logview/frontend/src/services/domains/user.ts

```diff
@@ -0,0 +1,20 @@
+import api from '../adapter';
+
+export default {
+	addUser: (user: any) => {
+		return api.makeRequest(`/api/users`, api.requestType.POST, user);
+	},
+	updateUser: (user: any) => {
+		return api.makeRequest(
+			`/api/users/${user._id}`,
+			api.requestType.PATCH,
```

Comment:
I think it's better to use PUT here:
The main difference between the PUT and PATCH method is that the PUT method uses the request URI to supply a modified version of the requested resource which replaces the original version of the resource whereas the PATCH method supplies a set of instructions to modify the resource.

Reply:
fixed